<!-- MHonArc v2.6.16 -->
<!--X-Subject: [bug #13442] Monte Carlo problem with model&#45;free.py and relax1.3.3 -->
<!--X-From-R13: nabalzbhf <@A&#45;DSBZK.W@HOZWR&#45;ORRDSEENtan.bet> -->
<!--X-Date: Tue, 28 Apr 2009 10:40:57 +0200 -->
<!--X-Message-Id: 20090428&#45;104052.sv0.62014@gna.org -->
<!--X-Content-Type: text/plain -->
<!--X-Head-End-->
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
   "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<title>[bug #13442] Monte Carlo problem with model-free.py and relax1.3.3 -- April 28, 2009 - 10:40</title>
<link rel="stylesheet" type="text/css" href="/archives-color-gna.css"> 
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
</head>
<body>
<!--X-Body-Begin-->
<!--X-User-Header-->
<!--X-User-Header-End-->
<!--X-TopPNI-->
<h2><img src="https://gna.org/images/gna.theme/mail.orig.png" width="48" height="48"
alt="mail" class="pageicon" />[bug #13442] Monte Carlo problem with model-free.py and relax1.3.3</h2>
<br />
<div class="topmenu">
<a href="../" class="tabs">Others Months</a> | <a href="index.html#00004" class="tabs">Index by Date</a> | <a href="threads.html#00004" class="tabs">Thread Index</a><br />
<span class="smaller">&gt;&gt;&nbsp;&nbsp;
[<a href="msg00003.html">Date Prev</a>] [<a href="msg00005.html">Date Next</a>] [<a href="msg00000.html">Thread Prev</a>] [<a href="msg00005.html">Thread Next</a>]
</div>

<!--X-TopPNI-End-->
<!--X-MsgBody-->
<!--X-Subject-Header-Begin-->
<h3><a name="header" href="#header">Header</a></h3>
<!--X-Subject-Header-End-->
<!--X-Head-of-Message-->
<ul class="headdata">
<li class="menuitem">
<em>To</em>: martinek@xxxxxxxxxxxxxxxxx, relax-devel@xxxxxxx</li>
<li class="menuitem">
<em>Date</em>: Tue, 28 Apr 2009 10:40:54 +0200</li>
<li class="menuitem">
<em>Message-id</em>: &lt;<a href="msg00004.html">20090428-104052.sv0.62014@gna.org</a>&gt;</li>
<li class="menuitem">
<em>References</em>: </li>
<li class="menuitem">
<em>User-agent</em>: Mozilla/5.0 (Windows; U; Windows NT 5.1; hu;	rv:1.9.0.6) Gecko/2009011913 Firefox/3.0.6</li>
</ul>
<!--X-Head-of-Message-End-->
<!--X-Head-Body-Sep-Begin-->
</div><!-- end headdata -->
<br />
<h3><a name="content" href="#content">Content</a></h3>
<div class="postedby">Posted by <strong>anonymous</strong> on April 28, 2009 - 10:40:</div>
<div class="msgdata">
<!--X-Head-Body-Sep-End-->
<!--X-Body-of-Message-->
<pre style="margin: 0em;">

URL:
  &lt;<a  rel="nofollow" href="http://gna.org/bugs/?13442">http://gna.org/bugs/?13442</a>&gt;

                 Summary: Monte Carlo problem with model-free.py and
relax1.3.3
                 Project: relax
            Submitted by: None
            Submitted on: Tuesday 04/28/2009 at 10:40 CEST
                Category: relax's source code
                Severity: 4 - Important
                Priority: 5 - Normal
                  Status: None
                 Privacy: Public
             Assigned to: None
         Originator Name: Tamas Martinek
        Originator Email: martinek@xxxxxxxxxxxxxxxxx
             Open/Closed: Open
         Discussion Lock: Any
                 Release: 1.3.3
        Operating System: GNU/Linux

    _______________________________________________________

Details:

Relax-1.3.3 from source on a RH9. The sample script model-free.py (attached)
runs nicely without monte carlo error estimaton. When the monte carlo step is
included, I get the following error:

Traceback (most recent call last):
  File &quot;/usr/local/bin/relax&quot;, line 422, in ?
    Relax()
  File &quot;/usr/local/bin/relax&quot;, line 131, in __init__
    self.interpreter.run(self.script_file)
  File &quot;/usr/local/relax/prompt/interpreter.py&quot;, line 270, in run
    return run_script(intro=self.__intro_string, local=self.local,
script_file=script_file, quit=self.__quit_flag,
show_script=self.__show_script, raise_relax_error=self.__raise_relax_error)
  File &quot;/usr/local/relax/prompt/interpreter.py&quot;, line 531, in run_script
    return console.interact(intro, local, script_file, quit,
show_script=show_script, raise_relax_error=raise_relax_error)
  File &quot;/usr/local/relax/prompt/interpreter.py&quot;, line 427, in
interact_script
    execfile(script_file, local)
  File &quot;model-free.py&quot;, line 78, in ?
    monte_carlo.initial_values()
  File &quot;/usr/local/relax/prompt/monte_carlo.py&quot;, line 165, in initial_values
    monte_carlo.initial_values()
  File &quot;/usr/local/relax/generic_fns/monte_carlo.py&quot;, line 269, in
initial_values
    init_sim_values()
  File &quot;/usr/local/relax/specific_fns/model_free/main.py&quot;, line 2938, in
sim_init_values
    object = getattr(cdp, object_name)
AttributeError: 'PipeContainer' object has no attribute 'chi2'

The chi2 is properly calculated during the first minimization before MC, and
dumped in the results file. The error message is reproduced with the test
data in the distribution(attached) too.




    _______________________________________________________

File Attachments:


-------------------------------------------------------
Date: Tuesday 04/28/2009 at 10:40 CEST  Name: model-free.py  Size: 3kB   By:
None

&lt;<a  rel="nofollow" href="http://gna.org/bugs/download.php?file_id=5721">http://gna.org/bugs/download.php?file_id=5721</a>&gt;
-------------------------------------------------------
Date: Tuesday 04/28/2009 at 10:40 CEST  Name: test.data.tgz  Size: 385B   By:
None

&lt;<a  rel="nofollow" href="http://gna.org/bugs/download.php?file_id=5722">http://gna.org/bugs/download.php?file_id=5722</a>&gt;

    _______________________________________________________

Reply to this item at:

  &lt;<a  rel="nofollow" href="http://gna.org/bugs/?13442">http://gna.org/bugs/?13442</a>&gt;

_______________________________________________
  Message sent via/by Gna!
  <a  rel="nofollow" href="http://gna.org/">http://gna.org/</a>



</pre>
<!--X-Body-of-Message-End-->
<!--X-MsgBody-End-->
<!--X-Follow-Ups-->
</div><!-- end msgdata -->
<br />
<h3><a name="related" href="#related">Related Messages</a></h3>
<div class="relateddata">
<ul><li><strong>Follow-Ups</strong>:
<ul>
<li><strong><a name="00006" href="msg00006.html">Re: [bug #13442] Monte Carlo problem with model-free.py and 	relax1.3.3</a></strong>
<ul><li><em>From:</em> Edward d'Auvergne</li></ul></li>
<li><strong><a name="00005" href="msg00005.html">[bug #13442] Monte Carlo problem with model-free.py and relax1.3.3</a></strong>
<ul><li><em>From:</em> Edward d Auvergne</li></ul></li>
</ul></li></ul>
<!--X-Follow-Ups-End-->
<!--X-References-->
<!--X-References-End-->
<!--X-BotPNI-->
</div><!-- end relateddata -->
<!-- NoBotLinksApartFromRelatedMessages -->

<!--X-BotPNI-End-->
<!--X-User-Footer-->
<!--X-User-Footer-End-->
<div class="footer">You are on the <a href="http://gna.org">Gna!</a> mail server.</div><br />
<div class="right">Powered by <a href="http://www.mhonarc.org">MHonArc</a>, Updated Tue Apr 28 14:01:13 2009</div>  
</body>
</html>
